README.md

```diff
@@ -1 +1,238 @@
-# motorcycle-rental-backend
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
- [ ]  You don't have any unit tests but the "test" folder still exists, which is used by another test framework that is not Rspec. Kindly consider deleting that folder if you are not using it. 👍